src/mono/wasm/debugger/tests/debugger-test/debugger-test.cs

```diff
@@ -2,7 +2,10 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 
 using System;
+using System.IO;
+using System.Runtime.Loader;
 using System.Threading.Tasks;
+
```

Comment:
Um there are tests depending on exact line numbers in this file, so this will likely break them. My suggested code was just trying to get this working. Maybe you could use FQN for the new types being used here?

The test code itself should be fine. Also, could you run `make -C src/mono/wasm run-debugger-tests` locally? We don't have that enabled on the CI yet.

Reply:
Fixed.